playpen/all_27.sh

```diff
@@ -0,0 +1,17 @@
+for repo in pulp pulp_puppet pulp_rpm
+do
+    pushd /home/vagrant/devel/$repo
+    git checkout 2.7-dev
+    find ~/devel -name "*.pyc" -delete
+    sudo ./pulp-dev.py -I
```

Comment:
How about using switches like `--install` instead of `-I`? These scripts will be read by many people in many different contexts, and I imagine some extra verbosity will help avoid confusion. Terseness is great when you're typing out things at a shell by hand, and that isn't happening here.